kotlin-react-dom-legacy/README.md

```diff
@@ -1,13 +1,12 @@
-# [archived]
-
----
-
 [![Maven Central](https://img.shields.io/maven-central/v/org.jetbrains.kotlin-wrappers/kotlin-react-dom-legacy)](https://mvnrepository.com/artifact/org.jetbrains.kotlin-wrappers/kotlin-react-dom-legacy)
 
 ## kotlin-react-dom-legacy
 
 Kotlin wrapper for the React DOM library. The major version number of this wrapper matches that of React DOM itself.
 
+> [!WARNING]
+> `kotlin-react-dom-legacy` is deprecated
```

Comment:
`kotlin-react-legacy` and `kotlin-react-dom-legacy` - 2 separate subprojects

`kotlin-react-legacy` README update also required

Reply:
Done 👍